src/modal/alert-modal.interface.ts

```diff
@@ -44,6 +44,12 @@ export interface AlertModalData {
 	 * Array of `ModalButton`s
 	 */
 	buttons?: Array<ModalButton>;
+	/**
+	 * Specify whether the modal contains scrolling content. This property overrides the automcatic detection of the existance of scrolling
+	 * content. Set this property to `true` to force overflow indicator to show up or to `false` to force overflow indicator to disappear.
+	 * It is set to `null` by default which indicates not to override automatic detection.
+	 */
```

Comment:
Typos and line length.

```suggestion
	/**
	 * Specify whether the modal contains scrolling content. This property overrides the automatic detection
	 * of the existence of scrolling content. Set this property to `true` to force overflow indicator to
	 * show up or to `false` to force overflow indicator to disappear.
	 * It is set to `null` by default which indicates not to override automatic detection.
	 */
```